### ISSUE TITLE
docs: 📚 describe link usage for sd-button

### ISSUE THREAD

**Issue by mariohamann** [solid-design-system/solid#1421]
## Description

@matuzo pointed out, that the usage of a button as a link (using `href`, `download` etc.) is completely missing in the docs. The danger is high that people use the button and do evil JavaScript things to imitate behaviour of a link. This would harm a11y and is super bad practice. We therefore have to add information in the docs how to use the button as a link.

Here's a great example for that: https://shoelace.style/components/button#link-buttons

## Where to put the documentation

Button

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by mariohamann** [comment]
Okay. Uhm. https://solid-design-system.fe.union-investment.de/docs/?path=/docs/components-sd-link--docs#as-link

We have it now. @MartaPintoTeixeira Is it in design as well? 😆

**Comment by mariohamann** [comment]
I was stupid. I linked the "sd-link" above. I optimized the text slightly in my PR for `sd-link` and it has to be added to `sd-button` as well. Could you take this over, @MartaPintoTeixeira ?

https://github.com/solid-design-system/solid/pull/2083

**Comment by MartaPintoTeixeira** [comment]
Changes done in figma for button as well and link copy updated as well.
I have approved the other PR already https://github.com/solid-design-system/solid/pull/2083
